app/components/recirculation-prefooter.js

```diff
@@ -39,7 +39,7 @@ export default Component.extend(
 
     listRendered: null,
     isContLangEn: equal('wikiVariables.language.content', 'en'),
-    displayLiftigniterRecirculation: and('isContLangEn', 'applicationWrapperVisible'),
+    displayLiftigniterRecirculation: and('isContLangEn', 'applicationWrapperVisible') && not('wikiVariables.enableFeedsAndPosts'),
```

Comment:
I think it won't work. AFAIK you can't connect computed properties with logical operators. It should be something like:
```
isFeedsAndPostsDisabled: not('wikiVariables.enableFeedsAndPosts'),
displayLiftigniterRecirculation: and('isContLangEn', 'applicationWrapperVisible', 'isFeedsAndPostsDisabled'),
```

Reply:
That's disappointing. It would be ideal to check that wiki variable for the `displayLiftigniterRecirculation` property. Ember seems pretty limited in that regard.

Reply:
That is actually not true. As in React you have to install some stuff to be productive, same thing happens in Ember world as well.
There is a great tool https://github.com/kellyselden/ember-awesome-macros we're just not using it currently.